plugins/inputs/gnmi/gnmi.go

```diff
@@ -6,6 +6,7 @@ import (
 	_ "embed"
 	"errors"
 	"fmt"
+	"google.golang.org/grpc/keepalive"
```

Comment:
Can you move this out of the std lib imports and above the `"google.golang.org/grpc/metadata"` import on line 16?

Reply:
sure.